RadarSDK/Include/RadarLocationPermissionStatus.h

```diff
@@ -18,33 +18,39 @@
  The type of location permissions state.
 */
 typedef NS_ENUM(NSInteger, RadarLocationPermissionState) {
-    NoPermissionGranted,
-    ForegroundPermissionGranted,
-    ForegroundPermissionRejected,
-    ForegroundPermissionPending,
-    BackgroundPermissionGranted,
-    BackgroundPermissionRejected,
-    BackgroundPermissionPending,
-    PermissionRestricted,
+    NoAuthorization,
+    ForegroundAuthorized,
+    ForegroundFullAccuracyDenied,
+    ForegroundAuthorizationDenied,
+    ForegroundAuthorizationRequestInProgress,
+    BackgroundAuthorized,
+    BackgroundFullAccuracyDenied,
+    BackgroundAuthorizationDenied,
+    BackgroundAuthorizationRequestInProgress,
+    AuthorizationRestricted,
     Unknown
```

Comment:
@lmeier here is my proposed renaming scheme to make our vocabulary more similar to that of Apples.
For reference here is Apple's docs for the new fields: https://developer.apple.com/documentation/corelocation/clservicesessiondiagnostic?changes=latest_minor&language=objc